npm/npm_import.bzl

```diff
@@ -323,10 +327,18 @@ def npm_translate_lock(
             ```
             lifecycle_hooks_execution_requirements: {
                 "*": ["requires-network"],
-                "@foo/bar": ["no-sandbox"],
+                "@foo/bar": ["no-remote-exec"],
             }
             ```
 
+        lifecycle_hooks_sandboxed: A list of npm packages for which lifecycle hooks, if they exist,
```

Comment:
I liked how the old way was just syntax sugar for execution requirements. I'm not clear why reversing the boolean is better.

Reply:
The better default for lifecycle hooks is `no-sandbox` but the old way there was no way to opt-in a package or two. Hence the flip. But I'm open to suggestions if you have a better idea for selective sandbox opt-in

Reply:
I think lifecycle_hooks_execution_requirements should express it all, the others are syntax sugar

Let's see if I can work this through...

`lifecycle_hooks_execution_requirements` default value is `{"*": ["no-sandbox"]}` for performance.

```
lifecycle_hooks_execution_requirements: {
    "*": ["requires-network", "no-sandbox"],
    "@foo/bar": ["no-remote-exec"],
}
```

Is how you'd allow all of them to use network and disable remote exec for one of them.

Can we deprecate `lifecycle_hooks_no_sandbox` and not replace it with anything?

Reply:
The trouble is there is no `sandbox` flag. If I set `"*": ["no-sandbox"],` I have no way to express _not_ setting `no-sandbox` on just one package that needs the sandbox.